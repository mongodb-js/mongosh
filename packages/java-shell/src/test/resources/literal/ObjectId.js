// command checkResultClass dontReplaceId
new ObjectId('5ebaf064121756574a9767cf')
// command checkResultClass dontReplaceId
ObjectId('5ebaf064121756574a9767cf')
// command checkResultClass
new ObjectId()