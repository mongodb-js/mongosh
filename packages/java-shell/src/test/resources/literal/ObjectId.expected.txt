ObjectIdResult: 5ebaf064121756574a9767cf
ObjectIdResult: 5ebaf064121756574a9767cf
ObjectIdResult: <ObjectID>